test/uk/gov/hmrc/pushpullnotificationsgateway/connectors/ProxiedHttpClientSpec.scala

```diff
@@ -19,13 +19,13 @@ package uk.gov.hmrc.pushpullnotificationsgateway.connectors
 import akka.actor.ActorSystem
 import org.scalatest.{Matchers, OptionValues, WordSpec}
 import org.scalatestplus.play.guice.GuiceOneAppPerSuite
-
 import play.api.inject.guice.GuiceApplicationBuilder
 import play.api.libs.ws.{WSClient, WSRequest}
 import play.api.{Application, Configuration}
 import uk.gov.hmrc.play.audit.http.HttpAuditing
+import util.HmrcSpec
 
-class ProxiedHttpClientSpec extends WordSpec with Matchers with GuiceOneAppPerSuite with OptionValues {
+class ProxiedHttpClientSpec extends HmrcSpec with GuiceOneAppPerSuite with OptionValues {
```

Comment:
MS and AMS are not required (HMRCSpec includes them). Appreciate this is not your code.

Reply:
Done